src/app/pages/home/containers/home-page.component.ts

```diff
@@ -35,16 +35,15 @@ import { TasksService } from '../services/tasks.service'
 })
 export class HomePageComponent {
   tasks: Promise<Task[]>
-  nextTask: Task = DefaultTask
-  showCalendar: boolean = false
-  showCompleted: boolean = false
-  showNoTasksToday: boolean = false
-  tasksProgress: TasksProgress
-  startingQuestionnaire: boolean = false
+  nextTask: Task
+  showCalendar = false
+  showCompleted = false
+  showNoTasksToday = false
+  tasksProgress: TasksProgress = { numberOfTasks: 1, completedTasks: 0 }
```

Comment:
Why don't we initialise with 0, 0 ?

Reply:
For the updateProgress() here: https://github.com/RADAR-base/RADAR-Questionnaire/pull/230/files/67f9adffdb241fdb4e551ead00fe41abef05446f#diff-e463b96da05f7f4d95a1d46290d9124fR45